README.md

```diff
@@ -38,8 +72,9 @@ By default, this package looks for your Salesforce data in the `salesforce` sche
 config-version: 2
 
 vars:
+    salesforce_database: your_database_name    
     salesforce_schema: your_schema_name
-    salesforce_database: your_database_name
+    salesforce__<default_source_table_name>_identifier: your_table_name
```

Comment:
Can we actually make this its own section within the README? A user doesn't **need** to configure this, but they may need to configure the database and schema variables.